src/vcf/validator.cpp

```diff
@@ -20,6 +20,12 @@ namespace ebi
 {
   namespace vcf
   {
+    bool validate(const std::vector<char> &firstLine,
```

Comment:
If `build_parser`, `is_valid_vcf_file` and `detect_version` are declared in the .hpp file, shouldn't these other two methods also be for consistency?